styles/globals.css

```diff
@@ -1,3 +1,53 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.exam-passed {
+    color: #10B981;
+}
+
+.exam-failed {
+    color: #C02727;
+}
+
+.exam-finished-success {
+    color: #10B981;
+    text-align: center;
+    font-size: 40px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 154.688%; /* 61.875px */
+}
+
+.exam-finished-fail {
+    color: #C02727;
+    text-align: center;
+    font-size: 40px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 154.688%; /* 61.875px */
+}
+
+.exam-finished-text {
+    color: #FFF;
+    text-align: center;
+    font-size: 24px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 154.688%; /* 37.125px */
+}
+
+.points-bg {
+    background-image: url('./../src/images/POINTS.png');
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+    -o-background-size:50% auto;
+    -webkit-background-size:50% auto;
+    -moz-background-size:50% auto;
+    background-size:50% auto;
+}
+
+.points-font {
+    font-size: 48px;
+}
```

Comment:
I would suggest using Tailwind to style instead of using those classes.

Reply:
could you refactor this in a new PR?